commons-numbers-complex-arrays/src/main/java/org/apache/commons/numbers/complex/arrays/ComplexList.java

```diff
@@ -177,6 +203,63 @@ public Complex set(int index, Complex element) {
         return oldValue;
     }
 
+    /**
+     * Replaces the complex element's real part at the specified position
```

Comment:
change `element` to `number`. Element is for a generic list; since this is typed to Complex then all element occurrences should be changed to `complex number`